### PR TITLE
Hello CDB namespace

### PR DESCRIPTION
**Pull request by xavijam**
Things DONE:
- [x] Changed styles, added Namespace.
- [x] Reviewed js files.
- [x] Reviewed tests.
- [x] Reviewed styleguides.
- [x] Reviewed js templates.

Pending review:
- ~~Infowindows (with @piensaenpixel).~~ https://github.com/CartoDB/cartodb.js/issues/899
- [x] Dashboard info styles.
- [x] Change histogram elements with new CDB thing.
- [x] Change Icon naming, they should be CDB-Icon-whatever.

Help:
- [x] Review SCSS-lint regexp (@javierarce please :) ).
- [x] Review widgets in more examples, checking that everything works as expected.

Notes:
- I've moved shapes stylesheet to common folder, I feel it could be used in other places.
- Not sure about moving text elements too, what do you think?. -> Answer: no for the moment
- What to do with chart elements like ```.Axis```, ```.Handle```,... etc? -> Answer: yes

Opinions?
@viddo @javierarce @alonsogarciapablo @piensaenpixel

### CODE REVIEW

**Comment by javierarce** [comment]
The [regexp looks fine](http://regexr.com/3cb18). It will accept `cdb-logo` as a valid rule, but I think that the cost of ruling this out is costly so I'd leave it like it's now.

**Comment by xavijam** [comment]
@javierarce could you check that histogram works as expected with these changes?

**Comment by javierarce** [comment]
Sure, I'll do it.

**Comment by xavijam** [comment]
And what do you think about moving or not the chart elements?

**Comment by javierarce** [comment]
I think we should move them and (if possible) try to rename them to follow the new convention.

File: src/geo/ui/dashboard-info-view.tpl [hunk 1]
```diff
@@ -1,56 +1,36 @@
-<div class="Dashboard-info-header">
-  <div class="Dashboard-info-logo">
-    <i class="CDBIcon CDBIcon-Cartofante"></i>
+<div class="CDB-Dashboard-info-header">
+  <div class="CDB-Dashboard-info-logo">
+    <i class="CDB-Icon CDB-Icon-Cartofante"></i>
```
Comment (piensaenpixel) [inline src/geo/ui/dashboard-info-view.tpl]: CDB-Icon-cartofante?

**Comment by javierarce** [comment]
@xavijam: histogram works like a charm :tm:

File: src/geo/ui/dashboard-info-view.tpl [hunk 2]
```diff
@@ -1,56 +1,36 @@
...
   </div>
-  <div class="Dashboard-info-actions">
-    <a href="#" class="Dashboard-info-actions-link js-toggle-view-link"></a>
+  <div class="CDB-Dashboard-info-actions">
```
Comment (piensaenpixel) [inline src/geo/ui/dashboard-info-view.tpl]: CDB-Dashboard-infoActions?

**Pull request merged**